File: CasinoAPI/CasinoAPI/Models/Tranzactie.cs
using System;

namespace CasinoAPI.Models
{
    public class Tranzactie
    {
        public int IDTranzactie { get; set; }

        public int UserId { get; set; }  // FK legat de User.Id

        public decimal Suma { get; set; }

        public string TipTranzactie { get; set; }

        public DateTime DataTranzactie { get; set; }

        public User User { get; set; }  // navigare către User
    }
}

File: CasinoAPI/Frontend/src/types/Tranzactie.ts
export interface Tranzactie {
  IDTranzactie: number;
  IDUtilizator: number;
  Suma: number;
  TipTranzactie: string; // ex: "Depunere", "Retragere"
  DataTranzactie: string; // ISO string
}
